Packages/MIES/MIES_DataConfigurator.ipf

```diff
@@ -1449,12 +1444,16 @@ static Function DC_FillDAQDataWaveForDAQ(string device, STRUCT DataConfiguration
 		if(config[i][%DAQChannelType] == DAQ_CHANNEL_TYPE_TP)
 			tpAmp = s.DACAmp[i][%TPAMP] * s.gains[i]
```

Comment:
low prio: I noticed that this line can be moved out of the DAQChannelType if block.

Reply:
Done.